client/my-sites/site-settings/site-visibility/launch-site.jsx

```diff
@@ -85,6 +84,13 @@ const LaunchSite = () => {
 
 	const LaunchCard = showPreviewLink ? CompactCard : Card;
 
+	// TODO: replace with actual value whether the site is a development site
+	const urlParams = new URLSearchParams( window.location.search );
+	const isDevelomentSite = urlParams.get( 'referer' ) === 'a4a-dashboard';
```

Comment:
Just noting a spelling error `isDevelomentSite` (missing `p`) which can be rectified in the follow PR later.

Reply:
Great catch! Thank you, Yan. I will just quickly fix it as I just cannot look at that typo. 😅